ddl/ddl_worker.go

```diff
@@ -393,6 +393,11 @@ func (d *ddl) runDDLJob(t *meta.Meta, job *model.Job) (ver int64, err error) {
 
 		job.Error = toTError(err)
 		job.ErrorCount++
+
+		metrics.DDLJobErrCounter.WithLabelValues("failed").Inc()
```

Comment:
If there is only one job error, there may be a repeat count.

Reply:
What do you mean?

Reply:
The counter has been increasing. `metrics` easy to spot error job.